From c266ec66d5662bea636b401fe74edfa10005271e Mon Sep 17 00:00:00 2001
From: zhuw <zhuw@codeaurora.org>
Date: Tue, 30 Jan 2018 13:49:15 +0800
Subject: [PATCH 24/67] SDGallery:Fix monkey FC when url from pick activity is
 null

if url in data of activityResult is null, don't load image

Change-Id: I103d1d431075483ef7cdce1a2bbd6459da69afdc
---
 src/com/android/gallery3d/filtershow/FilterShowActivity.java | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/com/android/gallery3d/filtershow/FilterShowActivity.java b/src/com/android/gallery3d/filtershow/FilterShowActivity.java
index 5c289ce02..7fad95601 100755
--- a/src/com/android/gallery3d/filtershow/FilterShowActivity.java
+++ b/src/com/android/gallery3d/filtershow/FilterShowActivity.java
@@ -2549,7 +2549,9 @@ DialogInterface.OnDismissListener, PopupMenu.OnDismissListener{
         if (resultCode == RESULT_OK) {
             if (requestCode == SELECT_PICTURE) {
                 Uri selectedImageUri = data.getData();
-                startLoadBitmap(selectedImageUri);
+                if (selectedImageUri != null) {
+                    startLoadBitmap(selectedImageUri);
+                }
             } else if (requestCode == SELECT_FUSION_UNDERLAY) {
                 Uri underlayImageUri = data.getData();
                 // find fusion representation
-- 
2.17.1

